pkg/flytek8s/client.go

```diff
@@ -37,23 +38,14 @@ func RemoteClusterConfig(host string, auth runtimeInterfaces.Auth) (*restclient.
 	}, nil
 }
 
-func GetRestClientConfigForCluster(cluster runtimeInterfaces.ClusterConfig) (*restclient.Config, error) {
-	kubeConfiguration, err := RemoteClusterConfig(cluster.Endpoint, cluster.Auth)
-
-	if err != nil {
-		return nil, err
-	}
-	logger.Debugf(context.Background(), "successfully loaded kube configuration from %v", cluster)
-	return kubeConfiguration, nil
-}
-
 // Initializes a config using a variety of configurable or default fallback options that can be passed to a Kubernetes client on
 // initialization.
 func GetRestClientConfig(kubeConfig, master string,
 	k8sCluster *runtimeInterfaces.ClusterConfig) (*restclient.Config, error) {
 	var kubeConfiguration *restclient.Config
 	var err error
 
+	kubeClientConfig := &config.GetConfig().KubeClientConfig
 	if kubeConfig != "" {
```

Comment:
Can we rename kubeConfig to kubeConfigPathString or something  specific because we have too many variations of kubeConfig , kubeConfiguration , kubeClientConfig etc

Reply:
Thanks for pointing this out, I thought the same thing!